ui/desktop/CHANGELOG.md

```diff
@@ -2,6 +2,17 @@
 
 Canonical reference for changes, improvements, and bugfixes for Boundary Desktop.
 
+## v1.7.0 [CLI 0.14.0] (2023.10.09)
+
+### New
+- Upgrade CLI to v0.14.0 ([PR]())
+- Update Connection workflow ([PR](https://github.com/hashicorp/boundary-ui/pull/1919))
+- Add Embedded Terminal ([PR](https://github.com/hashicorp/boundary-ui/pull/1884))
+
+### Bug Fixes
+- Update Electron to v26.x ([PR](https://github.com/hashicorp/boundary-ui/pull/1904))
```

Comment:
Is this considered a bug fix? I don't think we normally include these upgrades?

Reply:
Yeah good point. I added it because of the security issue but it isn't a bug fix. I will remove.